config/features.yml

```diff
@@ -184,6 +184,10 @@ features:
     actor_type: user
     description: When enabled, claims status tool uses the new claim details design
     enable_in_development: true
+  cst_use_dd_rum:
+    actor_type: user
+    description: When enabled, claims status tool uses DataDog's Real User Monitoring logging
+    enable_in_development: false
```

Comment:
I would set this to true (it's set to true for all of our other feature toggles)

Reply:
@jerekshoe The reason I didn't is that we're only supposed to test this in staging, and not supposed to turn it on in local environments (per the RUM guide Robin wrote): https://github.com/department-of-veterans-affairs/va.gov-team/blob/main/platform/analytics/setup-real-user-monitoring.md

Reply:
kk